package-lock.json

```diff
@@ -232,6 +232,7 @@
       "version": "1.3.5",
```

Comment:
Please add the package-lock.json file in gitignore

Reply:
The package-lock.json file is meant to be commited.
Please see https://docs.npmjs.com/files/package-lock.json